src/components/shown-data.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import ProgressBar from 'react-bootstrap/ProgressBar';
+import { MyStore } from '../types';
+interface shownDataProps {
+  data: any,
+}
+
+
+const ShownData: React.FC<shownDataProps> = (props)  => {
+  const now = 100;
+  const progressInstance = <ProgressBar now={now} label={`${now}%`} />;
+  return (
+    <div style={{width:'250px'}}>{progressInstance}</div>
```

Comment:
It's a bad practice to mix representation styles and code. Consider moving all the styling into css and assign className to a relevant components

Reply:
I know this. It was the only test and I forgot to delete it.